pages/linux/etckeeper.md

```diff
@@ -0,0 +1,24 @@
+# etckeeper
+
+> Track system configuration files in git.
+> More information: http://etckeeper.branchable.com/.
+
+- Initialize: Run once from /etc to initialize a new repo, and add package manager hooks for automatic commits when installing and removing packages:
```

Comment:
I think we should keep it short. No need to have a prefix "Initialize: ". And does the user need to add the package manager hooks or is it done automatically ?

Reply:
It's done automatically for most common package managers (though I'm having a hard time finding a list of which package managers exactly). I'll shorten it.